int64 dist
int64 numeroMovements
---
bool isWeak
